example/pubspec.yaml

```diff
@@ -15,47 +19,44 @@ version: 1.0.0+1
 publish_to: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-
-  flutter_map: ">=0.11.0 <1.0.0"
   flutter_map_marker_cluster:
     path: ../
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^0.1.2
+  cupertino_icons: ^1.0.2
+  latlong2: ^0.8.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-
-
+dependency_overrides:
```

Comment:
i don't like it .. please wait that nullsafety branch will be merged to dependency project

Reply:
@lpongetti it has been published now: version 0.13.0 so this is ready to be updated

Reply:
@lpongetti resolved now :)